### ISSUE TITLE
Astro Partytown not resolved correctly with base path config in SSR node standalone

### ISSUE THREAD

**Issue by harrytran998** [withastro/astro#5424]
### What version of `astro` are you using?

lastest

### Are you using an SSR adapter? If so, which one?

node

### What package manager are you using?

yarn 3

### What operating system are you using?

Mac

### Describe the Bug

```js
export default defineConfig({
  output: 'server',
  base: '/ok',
  server: {
    port: 3000,
    host: '0.0.0.0',
  },
  adapter: node({
    mode: 'standalone',
  }),
  integrations: [
     ...,
    AstroPartytown({ 
      config: { debug: true, forward: ['dataLayer.push'] }
    }),
],
})
```

When running dev or server mode --> Display this error in terminal

`[serve]    404        /~partytown/debug/partytown-sw.js`

And display this error in the browser

`TypeError: Failed to register a ServiceWorker for scope ('http://localhost:3000/ok/~partytown/debug/') with script ('http://localhost:3000/ok/~partytown/debug/partytown-sw.js'): A bad HTTP response code (404) was received when fetching the script.`

### Link to Minimal Reproducible Example

This link stackblitz https://stackblitz.com/github/harrytran998/reproduction-astro-base-path?file=package.json  

This link to Github if case Stackblitz does not work https://github.com/harrytran998/reproduction-astro-base-path

### Participation

- [ ] I am willing to submit a pull request for this issue.

**Comment by matthewp** [comment]
fixed by #5425

**Comment by harrytran998** [comment]
@matthewp Do you have the plan to release this 😁?
